config/routes.rb

```diff
@@ -1,6 +1,14 @@
 Rails.application.routes.draw do
+  get 'users/index'
+  get 'users/show'
+  get 'posts/index'
+  get 'posts/show'
```

Comment:
It is not advisable to have the routes set twice. Also, these routes are not correctly set.

- [ ] Please consider removing these routes as you only need those below, which are correctly set.

Reply:
Noted. Thank you